### PR TITLE
pkp/pkp-lib#7521 Crossref deposit message inaccurate when DOIs registered manually

### PR DESCRIPTION
**Pull request by ewhanson**

### CODE REVIEW

File: schemas/doi.json [hunk 1]
```diff
@@ -38,6 +38,11 @@
 	  "validation": [
 		"in:1,2,3,4,5"
 	  ]
+	},
+	"registrationType": {
```
Comment (NateWr) [inline schemas/doi.json:42]: What about existing DOIs? Do they get a `registrationType` set during migration?

File: schemas/doi.json [hunk 2]
```diff
@@ -38,6 +38,11 @@
...
+	  "type": "string",
+	  "description": "The way in which the DOI was registered, e.g. marked manually, registered with agency, etc.",
+	  "apiSummary": true
```
[the remark this answers is not in the document]
Comment (NateWr) [reply schemas/doi.json:45]: I should mention that this only effects documentation. You'd need to apply a validation rule to enforce it. But having read the code, I see the range of values comes from plugins, so you may not be able to specify.

File: schemas/doi.json (hunk 1 above)
Comment (ewhanson) [reply schemas/doi.json:42]: I was wondering about this. This detail is known at the time of migration, but is effectively removed as part of the initial DOI migration process. Seeing as we haven't hit a release yet, would you have any objections to updating this migration step (`I7014_DoiMigration`)? Would also be able to differentiate between marked registered and registered via the plugin for existing DOIs at this stage, too.

Comment (NateWr) [reply schemas/doi.json:42]: Yeah it's perfectly fine to update a migration before it's released.

**Pull request merged**